exchange/response_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/google/webpackager/resource/preload"
 )
 
-func makeHTTPResponse(payload []byte) *http.Response {
+func makeHTTPResponse(t *testing.T, payload []byte) *http.Response {
```

Comment:
Revert: this function no longer needs testing.T.

Reply:
Done.